app/Middleware/CorrelationIdGetter.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+const { v4: uuidv4 } = use('uuid')
+
+class CorrelationIdGetter {
```

Comment:
Strictly speaking, this class is a Setter, since its job is actually to set it on the context.

Also, speaking of getting, I think you forgot to add the part where it checks the request for a Request-Id header before generating one. We likely won't use that, but for completeness' sake.

Reply:
Yeah. I thought we wouldn't be needing that and that's why I didn't add it